File: CodeEval/fizzbuzz.rb
for i in 1..100
	if i % 15 == 0
		puts i.to_s + " fizzbuzz"
	elsif i % 3 == 0
		puts i.to_s + " fizz"
	elsif i % 5 == 0
		puts i.to_s + " buzz"
	end
end

File: CodeEval/fizzbuzz.pl
#!/usr/bin/perl
use feature qw(say);
$var = "hello world";

for ($i = 0;$i < 15; $i++){
    #print $i;
    fizzbuzz($i);
}

sub fizzbuzz{

    my $num = @_;
    print $num; 
    if($num){
        return $num;
    }
    
    if($num % 15 == 0){
        return "FizzBuzz";
    }

    if($num % 5 == 0){
        return "Buzz";
    }

    if($num % 3 == 0){
        return "Fizz";
    }
}
